src2_object_detection/Box bounding_box
stereo_msgs/DisparityImage disparity_image
---
geometry_msgs/PointStamped point
